man/systemd.netdev.xml

```diff
@@ -1575,14 +1575,14 @@
         <term><varname>RouteTable=</varname></term>
         <listitem>
           <para>The table identifier for the routes to the addresses specified in the
-          <varname>AllowedIPs=</varname>. Takes the special value <literal>off</literal>, one of the
-          predefined names <literal>default</literal>, <literal>main</literal>, and
-          <literal>local</literal>, names defined in <varname>RouteTable=</varname> in
+          <varname>AllowedIPs=</varname>. Takes a negative boolean value, one of the predefined names
```

Comment:
My brain fails to parse this, sorry. How does a negative boolean value for the `RouteTable=` field look like?

Reply:
E.g. `RouteTable=no` or `RouteTable=off`.

Reply:
Actually, it looks spurious comparing with other `RouteTable=` settings in other sections, e.g. `[DHCPv4]`. Maybe, we should add, say `RouteToAllowedIPs=` boolean setting. WDYT?

Reply:
I really don't want to further add more knobs until there have been some wider design discussions.

But I agree the field names are not chosen in a very self-explaining fashion. Wireguard interface configuration doesn't have routing tables per se, but this flag will cause wireguard to add routes *derived from allowedIPs* (and specify the name of the routing table to add to).

So I'd probably name it the other way round - `AllowedIPsToRoutingTable`, but again, let's sit down and have these discussions (including @zx2c4, if that bridge isn't burnt down), before adding more knobs. This again would be a change in the interface.